src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ CHANGELOG
 -----
 
  * added `AppVariable::getToken()`
- * Deprecated the possibility to inject the Form Twig Renderer into the form
-   extension. Inject it on TwigRendererEngine instead.
+ * [BC BREAK] Deprecated the possibility to inject the Form `TwigRenderer` into the form extension. 
```

Comment:
Passing the argument still works until 4.0, the deprecation is technically valid on the method scope as calling/extending it won't break, hence the keep of the mention. What breaks is the behavior around it, and only when using the extension outside of the full stack framework.

Updated:

> Injecting the Form `TwigRenderer` into the `FormExtension` is deprecated and has no more effect.